.ci/deps.sh

```diff
@@ -99,3 +99,6 @@ fi
 # Tailor (Swift) commands
 curl -fsSL https://tailor.sh/install.sh | sed 's/read -r CONTINUE < \/dev\/tty/CONTINUE=y/' > install.sh
 sudo bash install.sh
+
+# FlawFinder will be confused by our python 3 venv
+sudo sed -i 's/env python/python2/' /usr/bin/flawfinder
```

Comment:
We enable both 3 and 2.7 in out pyenv. Check [line](https://github.com/coala-analyzer/coala-bears/pull/434/files/8b518fede94de64930cf2e8c366ed7c66ec29a4b#diff-5c64bb36a9fa177d5dfeb57b93e57a6cR50)